src/linear_algebra/nonsingular_inverse.lean

```diff
@@ -381,5 +381,19 @@ begin
     exact is_unit_unit (A.nonsing_inv_unit h), },
 end
 
+lemma nonsing_inv_left_right (B : matrix n n α) (h : A ⬝ B = 1) : B ⬝ A = 1 :=
+begin
+  have h' : is_unit B.det :=
```

Comment:
This looks like it could be extracted to an independent lemma.

Reply:
Agreed and done.